src/client.rs

```diff
@@ -252,7 +252,7 @@ impl Client {
     pub async fn push(
         &mut self,
         image_ref: &Reference,
-        layers: Vec<ImageLayer>,
+        layers: &[ImageLayer],
```

Comment:
Super nit: Because this is a bug fix PR, I'd prefer to avoid the public API break here.

However, this is probably a good API change to make, so not going to block on it as we are coming up on a new release anyway

Reply:
Good point, fixed